src/components/lib/sl/sl_bind_check.c

```diff
@@ -0,0 +1,68 @@
+#include <sl.h>
+#include <stddef.h>
+
+#define NUM_FIELDS 14
+
+long __CTOR_LIST__;
+long __INIT_ARRAY_LIST__;
+long __DTOR_LIST__;
+long __FINI_ARRAY_LIST__;
+long __CRECOV_LIST__;
+
+static char * field_names[NUM_FIELDS] = {
```

Comment:
Maybe copy and paste in a comment the `sl_thd` structure so that people see where this is coming from?